src/CodeGenerator/src/Definition/StructureMember.php

```diff
@@ -22,6 +22,10 @@ public function getOwnerShape(): StructureShape
 
     public function isRequired(): bool
     {
+        if ($this->getOwnerShape() instanceof ExceptionShape && \in_array(strtolower($this->data['_name']), ['code', 'message'], true)) {
```

Comment:
Looking at generated code, it looks like all occurrences are `messages` (code is never required)

Reply:
I think this is only because of what we have currently generated. I think in principal newly generated options could use code. I copied what was done for `isNullable`.